static/app/components/core/text/index.mdx

```diff
@@ -178,38 +178,34 @@ Control line height with the `density` prop: `compressed` (1.0), default (1.2),
   <Storybook.SideBySide vertical>
     <div style={{width: '300px', marginBottom: '16px'}}>
       <Text bold as="p">
-        {`Compressed density.`}
+        Compressed density.
+      </Text>
+      <Text density="compressed">
+        {`Compressed density text. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.`}
```

Comment:
@natemoo-re are we able to remove the expression on these longer lines as well?

Reply:
Fixed in [8afed62](https://github.com/getsentry/sentry/pull/96351/commits/8afed62b7dabb3b5ac4dec0c39d634740d9c0a81)!